{{cookiecutter.project_slug}}/server/{{cookiecutter.project_slug}}/core/urls.py

```diff
@@ -38,8 +38,8 @@
         contact=openapi.Contact(email="support@{{ cookiecutter.project_slug }}.com"),
         license=openapi.License(name="BSD License"),
     ),
-    public=False,
-    permission_classes=[permissions.IsAuthenticated],
+    public=True,
+    permission_classes=[permissions.AllowAny],
```

Comment:
I can understand the convenience of this, but I have doubts that this is a smart 'production-ready' default. I think it would be very easy for a dev to overlook this and accidentally expose API docs to the world when they didn't mean to.

Perhaps it can be open when `DEBUG=True` and hidden behind auth when `DEBUG=False` ? That would be consistent with Django's behavior of showing a detailed stack trace and other dev tools when `DEBUG=True`.

Reply:
hmmm yea good point. I've been bouncing back and forth on my thoughts for this. Often API docs are public by default (and I plan with a follow-on PR, maybe this PR here, to lock down the user detail endpoints by default).
That said, I could see many/most users wanting it only accessible for logged-in users.
Conflicted...

Reply:
I'm going to undo my change here. I think there is a version of the bootstrapper where it's focused on "I'm building an API" and settings like this should default to open. But that's currently a distant 3rd place priority.